collections/high_availability/roles/drbd/README.md

```diff
@@ -62,6 +131,38 @@ drbdadm status
 drbdmon
 ```
 
+Example:
+```
+[root@mycluster ~]# drbdadm status
+resource0 role:Primary
+  disk:UpToDate
+  bee-meta2 role:Secondary
+    peer-disk:UpToDate
+
+resource1 role:Primary
+  disk:UpToDate
+  bee-meta2 role:Secondary
+    peer-disk:UpToDate
+```
+
+
+
+
+To delete the configuration (**will delete all data on the DRBD devices**)
+run the following commands :
+
+```
+drbdadm down resource0;drbdadm invalidate resource0;drbdadm wipe-md resource0 && rm -f /etc/drbd.d/resource0.res
+drbdadm down resource1;drbdadm invalidate resource1;drbdadm wipe-md resource1 && rm -f /etc/drbd.d/resource1.res
+rm -f /etc/drbd.d/global_common.conf
+```
+
+## Documentations
```

Comment:
A very nice idea: adding documentation links into the READMEs 😃 I will do that from now too.